### PR TITLE
Upgrade to PHP7.2 

### PR DESCRIPTION
**Pull request by Lutacon**
I just upgrade the infrastructure to a recent version. I hope this helps.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
 
   nginx:
-    image: nginx:1.11-alpine
```
Comment (jorge07) [inline docker-compose.yml]: keep the minor, it quite important

**Comment by jorge07** [comment]
You'll need to add the platform version for doctrine 
https://github.com/jorge07/symfony-4-es-cqrs-boilerplate/blob/master/config/packages/doctrine.yaml#L17

**Comment by Lutacon** [comment]
I have it working on local. I'm gonna try to fix it tomorrow

**Comment by Lutacon** [comment]
Just removed the MySQL 8 support.

**Comment by jorge07** [comment]
Needs rebase

**Comment by jorge07** [comment]
As always, thanks for the contribution! 🎉

**Pull request merged**